paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationHandler.kt

```diff
@@ -490,7 +639,9 @@ internal class IntentConfirmationHandler(
         /**
          * Indicates the the handler is currently confirming a payment.
          */
-        data object Confirming : State
+        data class Confirming(
+            val contentVisible: Boolean,
+        ) : State
```

Comment:
We should do some re-modeling to represent pre-confirming vs confirming. I didn't want to do it in this PR because of how big it is already.